aries-site/src/pages/components/skeleton-loading.mdx

```diff
@@ -0,0 +1,33 @@
+import { Notification } from 'grommet';
```

Comment:
We'll need to update this file name from "skeleton-loading" to "skeleton" for the routing to work

Reply:
done